### ISSUE TITLE
Feature request: return of the /projectE reloadEmc command

### ISSUE THREAD

**Issue by nayfaan** [sinkillerj/ProjectE#2085]
**Tickets that do not conform to this template will be closed without comment**

Exact ProjectE version (do not say "latest", "latest on Curse", or similar): ProjectE-1.16.5-PE1.0.0B

Exact Forge version: forge-1.16.5-36.0.14

Feature request:
If possible, I would like to see the return of the /projectE reloadEmc command. I know that currently the /reload command reloads the emc file, but when using projectE with a large modpack, the /reload command also reloads a lot of other stuff on other mods as well. With a heavy weight modpack, each time I run the /reload command it will use a lot of time trying to reload settings from other mods as well. Currently, as projectE is just released of 1.16, a lot of other mods do not have inherit EMC values, and I find myself having needed to add EMC values to other items often. If /projectE reloadEmc can return as an option to only refresh the projectE emc settings without reloading other mods as well, that would save a lot of time on reloads. Please tell if this is possible. Thanks

#2018

**Comment by pupnewfster** [comment]
I don't think this is really possible/practical as the reloading system is very heavily tied to the datapack reload system now so as to properly be able to update when tags/recipes are updated.

**Comment by AngleWyrm10** [comment]
Also issue #2128 where /reload caused another mod's commands to be wiped out. [The reason behind it](https://github.com/itsmeow/QuickHomes/issues/8#issuecomment-861906351) 

> The issue is that FMLServerStartingEvent is still being used to register commands instead of RegisterCommandsEvent. This means that when vanilla nukes the existing DataPackRegistry and creates a new instance your commands are not present as they aren't being registered again to the new DataPackRegistry
